1-js/06-advanced-functions/02-rest-parameters-spread-operator/article.md

```diff
@@ -170,8 +170,7 @@ let arr2 = [8, 3, -8, 1];
 alert( Math.max(...arr1, ...arr2) ); // 8
 ```
 
```

Comment:
Please,

- The code above seems to contain an error conflict.
(Would you check the English website article (link above)?)

Reply:
Same.